README.md

```diff
@@ -61,14 +61,50 @@ Then, transform incoming params in your controller using `Decanter#decant`:
 
 ### Generators
 
-Decanter comes with generators for creating `Decanter` and `Parser` files:
+Decanter comes with custom generators for creating `Decanter` and `Parser` files:
+
+#### Decanters
 
 ```
 rails g decanter Trip name:string start_date:date end_date:date
+
+# Creates app/decanters/trip_decanter.rb:
+class TripDecanter < Decanter::Base
+  input :id, :integer
+  input :name, :string
+  input :start_date, :date
+  input :end_date, :date
+end
 ```
 
+#### Parsers
 ```
 rails g parser TruncatedString
+
+# Creates lib/decanter/parsers/truncated_string_parser.rb:
+class TruncatedStringParser < Decanter::Parser::ValueParser
+  parser do |value, options|
+    value
+  end
+end
+```
+
+See more details on using custom parsers [here](#custom-parsers).
```

Comment:
I'm not a huge fan of "here" links (https://granicus.com/blog/why-click-here-links-are-bad/ has some general rationale). An alternative could be something like, "Learn more about [using custom parsers](#custom-parsers)."

Reply:
Thanks for the link. Done!